crypto/openssl_common.go

```diff
@@ -5,7 +5,7 @@ package crypto
 import (
 	pb "github.com/libp2p/go-libp2p-core/crypto/pb"
 
-	openssl "github.com/spacemonkeygo/openssl"
+	openssl "github.com/libp2p/go-openssl"
```

Comment:
We forked this library as upstream isn't maintained. Our library adds:

* FreeBSD support.
* Key equality checking.

Reply:
Added motivation for forking to the README.md in libp2p/go-openssl, for our future selves: https://github.com/libp2p/go-openssl/commit/1934e456a08650fa43aefb673eb660b7cac8c0b4